kube-tools/latest/Dockerfile

```diff
@@ -32,6 +32,14 @@ RUN set -euExo pipefail && shopt -s inherit_errexit && \
     esac && \
     /tmp/download-file.sh "https://github.com/operator-framework/operator-registry/releases/download/v1.56.0/linux-${architecture}-opm" "${checksum}" | cat >/usr/local/bin/opm  && \
     chmod +x /usr/local/bin/opm && \
+    mkdir -p /go/src/github.com/operator-framework && \
+    cd /go/src/github.com/operator-framework && \
+    # TODO: replace to upstream once https://github.com/operator-framework/operator-sdk/pull/6978 is released \
```

Comment:
let's at least edit the existing tracker (https://github.com/scylladb/scylla-operator-images/issues/101) to mention both affected images, not to forget about replacing here